b2sdk/transfer/emerge/emerger.py

```diff
@@ -0,0 +1,119 @@
+######################################################################
+#
+# File: b2sdk/transfer/emerge/emerger.py
+#
+# Copyright 2020 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+import logging
+
+import six
+
+from b2sdk.utils import B2TraceMetaAbstract
+
+from b2sdk.transfer.emerge.executor import EmergeExecutor
+from b2sdk.transfer.emerge.planner.planner import EmergePlanner
+
+logger = logging.getLogger(__name__)
+
+
+@six.add_metaclass(B2TraceMetaAbstract)
+class Emerger(object):
+    """
+    Handle complex actions around multi source copy/uploads.
+
+    This class can be used to build advanced copy workflows like incremental upload.
```

Comment:
From the code I can see that this class
creates a plan and uses the right executor to execute that plan. But I had to read the entire class to understand this. It would be nice to have this in the docstring of the class :)